lib/Monitorix.pm

```diff
@@ -70,15 +70,25 @@ sub celsius_to {
 	return $celsius;
 }
 
-sub picz_js_link {
+sub img_element {
+	my %params = @_;
+	return "<img src='" . $params{config}->{url} . "/" . $params{config}->{imgs_dir} . $params{IMG} . "' border='0'>"
+}
+
+sub picz_a_element {
+	my %params = @_;
+	return "<a href=\"" . $params{config}->{url} . "/" . $params{config}->{imgs_dir} . $params{IMGz} . "\">" . img_element(config => $params{config}, IMG => $params{IMG}) . "</a>"
+}
+
+sub picz_js_a_element {
 	my %params = @_;
 
 	my $zoom = (uc($params{config}->{image_format}) eq "SVG") ? (4 / 3) : 1;
 
 	my $picz_width = POSIX::ceil($params{width} * $zoom);
 	my $picz_height = POSIX::ceil($params{height} * $zoom + 0.5);
 
-	return "      <a href=\"javascript:void(window.open('" . $params{config}->{url} . "/" . $params{config}->{imgs_dir} . $params{IMGz} . "','','width=" . $picz_width . ",height=" . $picz_height . ",scrollbars=0,resizable=0'))\"><img src='" . $params{config}->{url} . "/" . $params{config}->{imgs_dir} . $params{IMG} . "' border='0'></a>\n";
```

Comment:
The leading spaces here are intended to keep the HTML source code a good indention (where possible).